robottelo/ui/base.py

```diff
@@ -220,6 +242,31 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
             )
             return None
 
+    def wait_until_element_is_clickable(self,
+                                        locator,
+                                        timeout=12,
+                                        poll_frequency=0.5):
+        """Wrapper around Selenium's WebDriver that allows you to pause your
+        test until an element in the web page is present and can be clicked.
+
+        """
+        try:
+            element = WebDriverWait(
+                self.browser, timeout, poll_frequency
+            ).until(expected_conditions.element_to_be_clickable(locator))
+            self.wait_for_ajax(poll_frequency=poll_frequency)
+            if element.get_attribute('disabled') == u'true':
+                return None
+            return element
+        except TimeoutException as err:
+            self.logger.debug(
+                "%s: Timed out waiting for element '%s' to display or to be "
+                "clickable.",
+                type(err).__name__,
+                locator[1]
+            )
+            return None
```

Comment:
This is syntactically fine. But does it make sense to return `None` both when an element is disabled and when it fails to become available?